examples/modis-006/modis-10A1-006/MOD10A1.A2022029.h10v05.006.2022031045818/MOD10A1.A2022029.h10v05.006.2022031045818.json

```diff
@@ -67,45 +67,15 @@
     "hdf": {
       "href": "../../../../tests/data-files/MOD10A1.A2022029.h10v05.006.2022031045818.hdf",
       "type": "application/x-hdf",
-      "title": "hdf data",
-      "eo:bands": [
-        {
-          "name": "NDSI_Snow_Cover",
-          "description": "Gridded NDSI snow cover and data flag values."
-        },
-        {
-          "name": "NDSI_Snow_Cover_Basic_QA",
-          "description": "A general estimate of the quality of the algorithm result."
-        },
-        {
-          "name": "NDSI_Snow_Cover_Algorithm_Flags_QA",
-          "description": "Algorithm-specific bit flags set for data screens and for inland water."
-        },
-        {
-          "name": "NDSI",
-          "description": "Raw NDSI values (i.e. prior to screening)."
-        },
-        {
-          "name": "Snow_Albedo_Daily_Tile",
-          "description": "Daily snow albedo corresponding to the NDSI_Snow_Cover parameter."
-        },
-        {
-          "name": "orbit_pnt",
-          "description": "Pointer to the orbit of the swath mapped into each grid cell. For details, see Section 1.1.2 below."
-        },
-        {
-          "name": "granule_pnt",
-          "description": "Pointer for identifying the swath mapped into each grid cell. For details, see the section 'Using the granule_pnt' below."
-        }
-      ],
+      "title": "Source data containing all bands",
```

Comment:
Can't figure out how to comment on the file generally. It looks like you are placing bitfield "flags" in the properties. Interesting.

Reply:
Do you mean `file:values` on the asset, e.g. [here](https://github.com/stactools-packages/modis/blob/33d2f6868720aa11b5cda44fcbd719b2c4065895/examples/modis-061/modis-10A2-061/MOD10A2.A2022033.h09v05.061.2022042050729/MOD10A2.A2022033.h09v05.061.2022042050729.json#L158-L219)?

Reply:
No. It's an 006 thing, so no matter. I commented on the wrong file, I guess. See here: https://github.com/stactools-packages/modis/blob/issues/63-metadata-updates/src/stactools/modis/fragments/21A2/006/item-properties.json, which gets pushed into some example jsons: https://github.com/stactools-packages/modis/blob/issues/63-metadata-updates/examples/modis-006/modis-21A2-006/MOD21A2.A2005361.h10v04.006.2018286181506/MOD21A2.A2005361.h10v04.006.2018286181506.json

Reply:
Yeah those were leftover from the first pass at this package, I would say they're _not_ the recommended way of doing things.